Source/Library/UIColor+Hex.swift

```diff
@@ -0,0 +1,29 @@
+import UIKit
+
+public extension UIColor {
+    
+    convenience init(hex string: String) {
```

Comment:
I think we can get into some sticky situations if we start copying code around, even if it means having dependencies. Especially if the signature is exactly the same, I think it saves you very little and causes "hidden" logic. This might be fine in just one project but this approach does not scale very well if you were to follow this rule for all projects. To avoid breaking `Lightbox` when updating `Hue`, I would specify a version in the `Podfile`. I think that would be easily discovered when updating `Lightbox` and trying to build the project on Travis.

That's my two cents on the issue, hope it made sense 😎